catalog/security/catalog-security-ingestplugin/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD description="Ingest Security Plugin"
+         name="Ingest Security Plugin"
+         id="ddf.catalog.security.ingest.IngestPlugin">
+
+        <AD description="Roles/attributes required for ingest operations (create, update and delete). Example: {role=role1;type=type1}"
+            name="Required Attributes" id="permissionsString" required="true" type="String"
+            default="{http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=anonymous}"/>
```

Comment:
Instead of making this a String value, it could be a list of Strings which would reduce special syntax and parsing of this configuration value.